src/Config.js

```diff
@@ -111,6 +112,10 @@ export class Config {
     this.validateIdempotencyOptions(idempotencyOptions);
     this.validatePagesOptions(pages);
     this.validateSecurityOptions(security);
+
```

Comment:
Could you use the `this.validate...` pattern we started to implement to make this more clean?
You could just duplicate an existing pattern, like `validateSecurityOptions`.